test/MicroOrm.Dapper.Repositories.Tests/DatabaseFixture/MySqlDatabaseFixture.cs

```diff
@@ -11,7 +11,7 @@ public class MySqlDatabaseFixture : IDisposable
         
         public MySqlDatabaseFixture()
         {
-            Db = new MySqlDbContext("Server=localhost;Uid=root;Pwd=Password12!");
+            Db = new MySqlDbContext("Server=localhost;Uid=root;Pwd=itsme999");
```

Comment:
If you change the password here, you should change also in docker-compose.yml

Reply:
Sorry, i revert this.